cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -138,6 +140,11 @@ public Network convert() {
             throw new CgmesModelException("Data source does not contain EquipmentCore data");
         }
         Network network = createNetwork();
+        network.setProperty("CGMES_topology", cgmes.isNodeBreaker() ? "NODE_BREAKER" : "BUS_BRANCH");
```

Comment:
As I said before, I think that properties for a network is not the good solution. We will have issues through the merging view. A slight extension could be better (and documented).